test/client/docs_test.rb

```diff
@@ -463,6 +468,7 @@
     percolator2 = @index.percolator "2"
     response = percolator2.create :query => { :match => { :author => "defunkt" } }
     assert response["created"], "Couldn't create the percolator query"
+    @index.refresh
```

Comment:
Why all the `@index.refresh` statements?

Reply:
In my local env I found that the registered percolator queries weren't always visible when the section of the tests that submitted docs for percolation ran. Adding these made them work every time, but we could possibly do something more clever here to minimize refreshes-per-test? (`populate!` also calls a refresh etc.)